doc/source/cluster/cluster_under_construction/ray-clusters-on-vms/user-guides/launching-clusters/index.rst

```diff
@@ -6,7 +6,8 @@
 .. toctree::
     :maxdepth: 2
```

Comment:
This page has `<no title>`, can we fix it?

https://ray--27431.org.readthedocs.build/en/27431/cluster/cluster_under_construction/ray-clusters-on-vms/user-guides/launching-clusters/index.html